### PR TITLE
 Ongoing repairs sorted to the top

### LINKED ISSUE #314: New repairs are sorted to be last in ecctool
When using ecctool to fetch information ongoing repairs. The repairs which are currently running are display last when they should be first.
The reason for this is that we sort after completed time and non completed repairs have a -1 in that field.

### PR DESCRIPTION
**Pull request by jwaeab**
…ongoing repairs are at the top of the list. This will make limit more useful as well. (#314)

Fixes #314

### CODE REVIEW

File: ecchronos-binary/src/pylib/ecchronoslib/table_printer_v2.py [hunk 1]
```diff
@@ -89,8 +89,8 @@ def print_repairs(repairs, max_lines=-1):
 
 def print_schedule_table(schedule_table, schedules, max_lines):
 
-    sorted_schedules = sorted(schedules, key=lambda sorted_schedule: sorted_schedule.last_repaired_at_in_ms,
-                              reverse=True)
+    sorted_schedules = sorted(schedules, key=lambda x: (x.last_repaired_at_in_ms, x.repaired_ratio, x.status),
```
Comment (masokol) [inline ecchronos-binary/src/pylib/ecchronoslib/table_printer_v2.py:92]: I would prefer if status wasn't part of the sorting. It's a string so it's hard to predict how sorting will be performed.

File: ecchronos-binary/src/pylib/ecchronoslib/table_printer_v2.py [hunk 2]
```diff
@@ -100,7 +100,7 @@ def print_schedule_table(schedule_table, schedules, max_lines):
 
 def print_repair_table(repair_table, repairs, max_lines):
 
-    sorted_repairs = sorted(repairs, key=lambda sorted_repair: sorted_repair.completed_at, reverse=True)
+    sorted_repairs = sorted(repairs, key=lambda x: (x.completed_at, x.repaired_ratio, x.status), reverse=False)
```
Comment (masokol) [inline ecchronos-binary/src/pylib/ecchronoslib/table_printer_v2.py:103]: same as above.

File: ecchronos-binary/src/pylib/ecchronoslib/table_printer_v2.py (hunk 1 above)
Comment (jwaeab) [reply ecchronos-binary/src/pylib/ecchronoslib/table_printer_v2.py:92]: I'm not sure I follow this one. Sorting will be in descending order. The status is somewhat hardcoded throughout, so we couldn't just change it like that anyhow.

File: ecchronos-binary/src/pylib/ecchronoslib/table_printer_v2.py (hunk 2 above)
Comment (jwaeab) [reply ecchronos-binary/src/pylib/ecchronoslib/table_printer_v2.py:103]: Same as above.

**Review by masokol — APPROVED** [review]

**Pull request merged**